tools/remote_build/windows.bazelrc

```diff
@@ -0,0 +1,2 @@
+build --test_tag_filters=-exclude_windows
+build --build_tag_filters=-exclude_windows
```

Comment:
if this file is really needed, it seems to be misplaced: the name `remote_build/windows.bazelrc` kind of implies that using this configuration will run a windows RBE build (which is currently not the case).

Ideally, the windows RBE (and local bazel) runs will be run exactly in the same way as the linux  builds:
- tools/remote_build/manual.bazelrc would be used to spin up RBE builds manually
- tools/remote_build/kokoro.bazelrc would be used by kokoro jobs that run windows RBE builds.

I understand this might be hard to achieve from the very start, but if we need some intermediate configuration files, we should make that clear (at least with a comment).

Reply:
This PR is not for RBE, and I don't see any other place for .bazelrc files. There will be another PR for Windows RBE. This is an intermediate for enabling RBE. TODO's added for future PR's. Having everything in one massive PR is definitely not the most optimal.

Reply:
Yeah, I get this is not for RBE, that's why I'm complaining about adding windows.bazelrc file under "remote_build" (and I'm also trying to explain how the future configuration layout should look like and trying to make sure that this PR is aligned with that idea).
Just a nit: "not seeing any other place for .bazelrc files" doesn't mean that the bazelrc file automatically belongs here.
I'm probably fine with the windows.bazelrc file being here temporarily, but there needs to be a cleanup phase because I don't think the current layout of windows bazelrc files is the best one.

Reply:
As stated in the latest change, it's supposed to be merged inside rbe_common.bazelrc, currently that file is tailored for Linux, so that's the next step